README.md

```diff
@@ -1417,6 +1417,38 @@ Other Style Guides
       return name;
     }
     ```
+  <a name=variables--no-chain-assignment""></a><a name="13.5"></a>
+  - [13.5](#variables--no-chain-assignment) Don't chain variable assignments. eslint: [`one-var`](http://eslint.org/docs/rules/one-var.html)
```

Comment:
I figured since it was related, it would apply, but if you think it's misleading we can remove it.